packages/web/pages/loot/unbundle-success.tsx

```diff
@@ -0,0 +1,276 @@
+    `}
+  }
+  ul li {
+    font-size: 1.25em;
+    background-color: rgba(255, 255, 255, 0.125);
+    text-align: center;
+  }
+  a {
+    display: block;
+    padding: 1em;
+  }
+  .prefix,
+  .suffix {
+    color: #fdff6e;
+    font-size: 0.9em;
+  }
+  opacity: 0;
+  @keyframes scroll-in {
+    0% {
+      top: 80%;
+      opacity: 1;
+    }
+    100% {
+      top: -200em;
+      opacity: 1;
+    }
+  }
+  animation: scroll-in 60s linear 2s;
+`;
+
+const MastheadContainer = styled.div`
+  position: fixed;
+  top: 150%;
+  left: 50%;
+  transform: translate(-50%, -25%);
+  width: 80%;
+  display: flex;
+  justify-content: center;
+  ${media.tablet`
+    width: 50%;
+  `}
+  @keyframes scroll-in {
+    0% {
+      top: 100%;
+      opacity: 1;
+    }
+    25% {
+      top: 75%;
+      opacity: 1;
+    }
+    50% {
+      top: 50%;
+      opacity: 1;
+    }
+    75% {
+      top: 15%;
+      opacity: 1;
+    }
+    99% {
+      top: 15%;
+      opacity: 1;
+    }
+    100% {
+      top: 25%;
+      opacity: 0;
+      display: hidden;
+    }
+  }
+  animation: scroll-in 3s linear 0s;
+`;
+const AlertContainer = styled.div`
+  left: 50%;
+  opacity: 0;
+  position: fixed;
+  top: 15%;
+  transform: translate(-50%, -50%);
+  width: 80%;
+  ${media.tablet`
+    width: 50%;
+  `}
+  @keyframes appear {
+    0% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+  animation: appear 0.1s linear 3.1s 1 forwards;
+`;
+
+const UnbundleSuccess = () => {
+  const [gangstaParty, setGangstaParty] = useState(false);
+
+  const image = gangstaParty ? 'bridge_with_hustlers.png' : 'bridge_no_hustlers.png';
+
+  const ScreenSaver = styled.div`
+    position: absolute;
+    top: 0px;
+    left: 0px;
+    width: 100%;
+    height: 100%;
+    z-index: 0;
+    background: #000000 url('/images/hustler/${image}') center / cover no-repeat fixed;
+  `;
+
+  return (
+    <>
+      <Head title="Bridging and unbundling loot to Optimism" />
+      <ScreenSaver>
+        {gangstaParty && (
+          <>
+            <PlugContainer>
+              <Image src="/images/masthead/ogs.svg" alt="DOPE OGS" />
+              <ul>
+                {PLUGS.sort(() => Math.random() - 0.5).map((plug, index) => {
+                  return (
+                    <li key={`plug-${index}`}>
+                      <a href={plug.link} target={plug.name}>
+                        {plug.prefix ? <div className="prefix">&quot;{plug.prefix}&quot;</div> : ''}
+                        {plug.name}
+                        {plug.suffix ? <div className="suffix">&quot;{plug.suffix}&quot;</div> : ''}
+                      </a>
+                    </li>
+                  );
+                })}
+              </ul>
+            </PlugContainer>
+            <WebAmpPlayer />
+          </>
+        )}
+        {!gangstaParty && (
+          <>
+            <MastheadContainer>
+              <Image src={randomMast()} alt="Dope." />
+            </MastheadContainer>
```

Comment:
Can we pull some of this into common components? Maybe the success screen can be generic and take the message/call to actions as props?